src/editor/gltf/GLTFLoader.js

```diff
@@ -1446,9 +1446,41 @@ class GLTFLoader {
 
     if (materialDef.extensions) this.addUnknownExtensionsToUserData(material, materialDef);
 
+    this.applyTextureTransform(materialDef.emissiveTexture, material.emissiveMap);
+    this.applyTextureTransform(materialDef.pbrMetallicRoughness.occlusionTexture, material.aoMap);
+    this.applyTextureTransform(materialDef.pbrMetallicRoughness.baseColorTexture, material.map);
+    this.applyTextureTransform(materialDef.pbrMetallicRoughness.metallicRoughnessTexture, material.metalnessMap);
+    this.applyTextureTransform(materialDef.normalTexture, material.normalMap);
+
     return material;
   }
 
+  /**
+   * Applies a texture transform, if present, to the map definition. Requires
+   * the KHR_texture_transform extension.
+   */
+  applyTextureTransform(mapDef, map) {
+    if (!mapDef || !map || !mapDef.extensions || !mapDef.extensions["KHR_texture_transform"]) {
+      return;
+    }
+
```

Comment:
In the Three.js implementation, they clone the texture object and set `texture.needsUpdate = true` I think we'll want to do the same just to be sure. https://github.com/mrdoob/three.js/blob/dev/examples/js/loaders/GLTFLoader.js#L812

Reply:
I've updated to make it more similar to the ThreeJS loader.